website_crm_privacy_policy/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Website CRM privacy policy
+==========================
+
+This module was written to add a checkbox to contact us form for user privacy
+policy validation
+
+Privacy policy is provided by Website legal page addon (website_legal_page)
+
+Usage
+=====
+
+Website editors can change easily any text of these pages using website builder
```

Comment:
https://github.com/OCA/maintainer-tools/pull/108 for the inclusion of the repo ID